resources/views/layout-extern/index.blade.php

```diff
@@ -23,13 +23,45 @@
 			<div class="inner">
 				<span class="image"><img src="/fotoalbum/Publiek/Voorpagina/_resized/CSR_Delft.jpg" alt="Foto vereniging"/></span>
 				<div class="content">
-					<h2 class="major">C.S.R. Delft</h2>
-					<p>
+					<p class="major" style="padding-bottom: 1em;border-bottom: solid 4px #e59200;">
 						<em style="color:red;">
 							Vanwege de recente maatregelen omtrent de verspreiding van het coronavirus, is besloten per direct alle verenigingsactiviteiten af te lassen en is onze sociëteit gesloten. Als u hierover vragen heeft of het bestuur wil bereiken kunt u het bestuur mailen (bestuur@csrdelft.nl).<br>
 						</em>
 						<strong>Heb je interesse om lid te worden bij C.S.R.? Stuur een appje/maak een praatje met Maartje (06 3327 1913) of mail promocie@csrdelft.nl.</strong>
 					</p>
+
+					<div class="een-minuut">
+						<div id="hero">
+							<h2>C.S.R. IN 1 MINUUT</h2>
+							<div class="bb-video">
+								<iframe src="https://www.youtube.com/embed/AE8RE8e5qI4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+							</div>
+						</div>
+						<div class="sociaal">
+							<div class="youtube">
+								<p>Wil je zien hoe de vereniging in elkaar zit? Bekijk de serie 'Delft studie is maar de helft!'</p>
+								<div bb-video>
+									<iframe src="https://www.youtube.com/embed/a7hhtoo_kzY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+								</div>
+							</div>
+							<div class="instagram">
+								<p>Bekijk en volg ook ons instagram account voor meer van onze vereniging!</p>
+								<a href="https://www.instagram.com/csrdelft/">[Instagram foto]</a>
+							</div>
+						</div>
+
+						<p>
+							De OWee gaat er anders uitzien, voor meer informatie zie <a href="https://owee.nl">https://owee.nl/</a>.<br>
```

Comment:
Zie de `<a..>OWee website</a>` lijkt me hier beter. of anders `<a...>owee.nl</a>`

Reply:
Goede ja! Ook aangepast